core/client/fs/src/main/java/alluxio/client/block/stream/PacketOutStream.java

```diff
@@ -173,13 +173,6 @@ public void flush() {
     for (PacketWriter packetWriter : mPacketWriters) {
       packetWriter.flush();
     }
-
-    // Release the channel used in the packet writer early. This is required to avoid holding the
-    // netty channel unnecessarily because the block out streams are closed after all the blocks
-    // are written.
-    if (remaining() == 0) {
-      close();
-    }
```

Comment:
Currently it is 1024 which should be sufficient, ideally we want to move to commit per block which will fix this bottleneck.